src/store/Entity/EntityStripper.ts

```diff
@@ -0,0 +1,101 @@
+import Dictionary from '@/common/interfaces/Dictionary';
```

Comment:
Wikibase has a service called `EntityInitializer` which does basically the same thing. Conceptually, I find it odd that the `ENTITY_INIT` action takes a service emitting different parts of the entity instead of the entity object itself.

Here is my yes-model:
* rename this to `EntityInitializer`
* have a single method `getEntityFromJson( serializedEntity: string ): Entity` - `Entity` being an interface containing all the required fields
* pass the Entity object instead of the initializer into the action